enterprise/models.py

```diff
@@ -312,6 +312,13 @@ class Meta:
         on_delete=models.CASCADE
     )
 
+    disable_expiry_messaging_for_learner_credit = models.BooleanField(
+        verbose_name="Disable expiration messaging for learner credit",
+        default=False,
+        help_text=_("Automatically disabled. If unchecked, learners and admins will not receive email, "
```

Comment:
[clarification, sanity check]

>  If unchecked, learners and admins will not receive email

Should this say, "If **checked**, learners and admins will not receive email"? Only when the new config field is truthy/_enabled_ is the expiration messaging disabled? 🤔 

Similar concern for the "automatically disabled" bit. Does this mean the `disable_expiry_messaging_for_learner_credit` is disabled, with a double negative?

Reply:
First point. You're correct if checked they should not receive the following 👍🏽  good catch.

Second point, I can see how the double negative could seem confusing. Just wondering if removing this line would serve the same purpose so it becomes `If checked, learners and admins will not receive email, "
                    "and notifications on expiring learner credit plans.`